sd_jwt/__init__.py

```diff
@@ -0,0 +1,6 @@
+__version__ = "0.1.0"
+
+DEFAULT_SIGNIGN_ALG = "RS256"
```

Comment:
```suggestion
DEFAULT_SIGNING_ALG = "RS256"
```

Reply:
My bad, this will need a more accurate refactor, I'll do it, due to this I cant merge your suggestion right now.
thank you, nice shot

Reply:
done here: https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/b30cbc9bf43e003eb523df3989f0a1ea6e27cc73